records_mover/session.py

```diff
@@ -156,6 +158,19 @@ def _boto3_session(self) -> boto3.session.Session:
         else:
             return self.creds.boto3_session(self._default_aws_creds_name)
 
+    def set_stream_logging(self,
+                           app_name: str = 'records_mover',
+                           default_root_log_level: int = logging.INFO,
+                           stream: IO[str] = sys.stdout,
+                           fmt: str = '%(asctime)s - %(message)s',
+                           datefmt: str = '%H:%M:%S') -> None:
```

Comment:
Lots of arguments, but all are defaulted, so you can just call it and get some kind of sensible logging set up if you want.